lib/ripple/core_ext/indexes.rb

```diff
@@ -51,7 +51,7 @@ class DateTime
   def to_ripple_index(type)
     case type
     when 'bin'
-      utc.to_s(Ripple.date_format)
+      utc.to_datetime.to_s(Ripple.date_format)
```

Comment:
I'm not clear why this is done differently than https://github.com/spreedly/ripple/pull/5/files#diff-c5e5688bbbe445aa0e40371204050f51R122 . I don't have any strong opinion here; just confused.